sdk/storage/azdatalake/internal/generated_blob/zz_options.go

```diff
@@ -0,0 +1,1463 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// Code generated by Microsoft (R) AutoRest Code Generator. DO NOT EDIT.
+// Changes may cause incorrect behavior and will be lost if the code is regenerated.
+
+package generated_blob
+
+import "time"
+
+// AppendBlobClientAppendBlockFromURLOptions contains the optional parameters for the AppendBlobClient.AppendBlockFromURL
+// method.
+type AppendBlobClientAppendBlockFromURLOptions struct {
```

Comment:
This is a new file being added, which means these options were earlier part of some other file and now being removed from there ?

Reply:
Yes, these were part of models.go before, which have been moved to a new file